suites/suite_cufflinks_2_2_1/.shed.yml

```diff
@@ -0,0 +1,2 @@
+owner: devteam
```

Comment:
all of the devteam suites are like this. I was thinking/hoping we wouldn't even need suite folders if planemo was going to handle the decomposition...